common/script/libs/statsComputed.js

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash';
+import content from '../content/index';
+import splitWhitespace from '../libs/splitWhitespace';
+const $w = splitWhitespace;
+import * as statHelpers from '../statHelpers';
+
+module.exports = function statsComputed (user) {
+  let thisArr = $w('stats stats.buffs items.gear.equipped.weapon items.gear.equipped.armor items.gear.equipped.head items.gear.equipped.shield');
+  let computed = _.reduce(['per', 'con', 'str', 'int'], (m, stat) => {
```

Comment:
Almost everything in lodash can be written using native javascript. _.reduce() is pretty much the same as Array.prototype.reduce(). By the way, I prefer underscore library instead of lodash because the former is semantically cleaner. http://underscorejs.org/ In this particular case, using lodash or native js I think makes no difference.